src/atoms/cc-select.js

```diff
@@ -200,13 +201,15 @@ export class CcSelect extends LitElement {
         }
 
         select[disabled] {
-          background: #eee;
-          border-color: #eee;
+          background: var(--color-bg-neutral-disabled);
+          border-color: var(--color-bg-neutral-disabled);
+          color: var(--color-text-light);
+          opacity: 1;
```

Comment:
@hsablonniere changed this so that disabled inputs look the same.
You can check the preview to see the result, it is up to date.

Text is a little bit darker than what it was before since I removed opacity to make sure it remains contrasted enough. If you feel like it is too close to the readonly state, we can bring back opacity for now (since we have to deal with the disabled subject in general anyway).